application/src/main/java/org/togetherjava/tjbot/commands/tag/TagManageCommand.java

```diff
@@ -0,0 +1,233 @@
+package org.togetherjava.tjbot.commands.tag;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.events.interaction.ButtonClickEvent;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.exceptions.ErrorResponseException;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.CommandData;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import net.dv8tion.jda.api.interactions.components.Button;
+import net.dv8tion.jda.api.interactions.components.ButtonStyle;
+import net.dv8tion.jda.api.requests.ErrorResponse;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.AbstractCommand;
+import org.togetherjava.tjbot.tag.TagSystem;
+
+import java.awt.*;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class TagManageCommand extends AbstractCommand {
+    private static final String TAG_ID = "Tag id";
+    private static final String CONTENT = "content";
+    private static final String TAG_MINUS_ID = "tag-id";
```

Comment:
the names for those are pretty bad imo. prefer a name that explains its usage instead of the plain content. the `MINUS` triggers me, that should be removed definitely.

Reply:
sonarcloud was telling me to do those. I'll revert it.

Reply:
i  think u misunderstood me here. having fields is good. i just dont like the names u gave. sorry for being not precise enough.

Reply:
What should the names of the constants be then?